app/feature_defaults_unittest.cc

```diff
@@ -277,6 +277,7 @@ TEST(FeatureDefaultsTest, EnabledFeatures) {
 #if !BUILDFLAG(IS_ANDROID)
       &sharing_hub::kDesktopScreenshots,
 #endif
+      &features::kDesktopPWAsTabStripSettings,
```

Comment:
Nit: Can we keep this sorted by `namespace::feature`? It just helps to be able to quickly run down the list. Thanks!

Reply:
good call!